Sources/SessionTools/Base/Session.swift

```diff
@@ -31,7 +31,7 @@ open class Session<T: Codable> {
     }
     
     public var currentItem: T? {
-        guard let user = try? container.item(forIdentifier: storageIdentifier, jsonDecoder: decoder) else { return nil }
+        guard let user = ((try? container.item(forIdentifier: storageIdentifier, jsonDecoder: decoder)) as T??) else { return nil }
```

Comment:
The double optional feels weird here. Is there anything we can do to try and make this simpler?

Reply:
I am not a fan of it either but that's what the migrator recommended. I prefer the old way but I'll see if there is a simpler way.

Reply:
I changed it back and it seems to work.

Reply:
Actually, we may need to use the `swift_version` attribute inside of the Podspec. Looks like `.swift-version` file support is [going away](https://stackoverflow.com/a/54060826/4343618) in Cocoapods 1.7.0.

Reply:
Going to update the pod spec